Add details on blocking data using JSON Schema (#5522)
segmentio/segment-docs: src/protocols/tracking-plan/create.md

> info "Specifying data type"
> Property or trait data type should adhere to the [data types defined by JSON schema](https://json-schema.org/understanding-json-schema/reference/type.html){:target="_blank”}. Data type names must be lower-cased as specified in JSON schema. Datetime properties should be represented as a `string` type with [`format` keyword](https://json-schema.org/understanding-json-schema/reference/string.html#format){:target="_blank”} (for example: "format": "date-time").

> info "Blocking data"

> > JSON schema violation event blocking is ONLY supported in cloud-mode Destinations. See [here](https://segment.com/docs/protocols/enforce/schema-configuration/){:target="_blank”} for more information on blocking data.

Reviewer: ```suggestion
> JSON schema violation event blocking is **only** supported in cloud-mode Destinations. See [the Customize your schema controls docs](/docs/protocols/enforce/schema-configuration/) for more information on blocking data.
```
